packages/ui/src/lib/util.ts

```diff
@@ -0,0 +1,25 @@
+export const newEnterKeyHandler = (handler: (e: KeyboardEvent) => any) => {
+	const ENTER_KEY = 13;
+	return (e: KeyboardEvent) => {
+		if (e.keyCode === ENTER_KEY) {
```

Comment:
```suggestion
	return (e: KeyboardEvent) => {
		if (e.key === 'Enter') {
```

[MDN says](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) that the `keyCode` property is deprecated: use `key` instead (or `code` in cases where you really need to distinguish between physical keys rather than characters).